Pythonwin/pywin/test/_dbgscript.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: latin-1 -*-
```

Comment:
Is latin-1 necessary for the test, or can we just save this as the default utf-8? (Github is showing mojibake so just using utf-8 seems cleaner if it doesn't defeat the purpose of the test - and if it does, maybe a comment as to why?

Reply:
Yes, the non-default encoding detection by Pythonwins open-document is tested using this and those special chars right below. (_# Umlauts for encoding test : ..._  ; search "encoding" in the diff for the corresponding tests). And the default utf-8 encoding on the other hand is tested on one of the other py files.

When "View file" for that file is opened in Github from the diff/FilesChanged view, it shows it in corrected encoding in Github. Probably because then the Githubs encoding guess for web display is unambiguous with single file data.
An strange: when (after that?) I open again the Github diff view from this review comment here a 2nd time - it suddenly shows the same chars in right encoding even in the diff view!? So it seems the Githubs web renderer just uses the "git diff" output (binary data) and somehow smart-guesses the best encoding in mixed cases - and that guess may depend on (or evolve?) how and/or when its happening ...  Nothing essential inside git repo itself.